arduino_unit_tests/arduino_unit_tests.ino

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file arduino_unit_tests.ino
+ * @author Aryan Kalia
+ * @brief File containing the arduino unit tests for the cc1120_spi.cpp file.
+ * @version 0.1
+ * @date 2022-02-13
+ * 
+ */
+
+#include <SPI.h>
+#include "arduino_unit_tests.h"
+#include "cc1120_spi.h"
+#include "cc1120_unit_tests.h"
+#include "arduino_unit_tests.h"
```

Comment:
The arduino_unit_tests header is included twice here.

Reply:
Fixed, thanks